target/linux/ath79/dts/qca9558_tl-wr1043nd-v3.dts

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9558_tl-wr1043nd-v2.dts"
```

Comment:
include files always have the .dtsi file extensions. The qca9558_tl-wr1043nd-v2.dts should be renamed to something like qca9558_tl-wr1043nd.dtsi. Create a new qca9558_tl-wr1043nd-v2.dts which includes the dtsi file and only defines the compatible and model (as you have done for the v3).

dtsi files should only have common parts defined. In best case dts files shouldn't overwrite defines/setting made in dtsi files. If you need to do so, you should consider moving the setting from the dtsi file to the dts files.

Reply:
the targets are the same in hardware, only uboot needs another signature on the image and that is taken care in the image creation so they can be split and only the name to be changed